### PR TITLE
added styles to the card header part

### PR DESCRIPTION
**Pull request by shewood**
### Description

List of proposed changes:
Updated the styling on the benefit card header.
-
-

### CODE REVIEW

File: components/molecules/BenefitCardHeaderSummary.js [hunk 1]
```diff
@@ -7,18 +7,20 @@ export default function BenefitCardHeaderSummary(props) {
...
-      {!typesWithLinks.find((t) => t === props.summary.type) ? null : (
-        <p>
-          <Link
-            text={t[props.summary.type].linkText}
-            href={t[props.summary.type].link}
-          />
-        </p>
-      )}
+    <div className="w-full ">
+      <div className="w-full  ">
```
Comment (s-laugh) [inline components/molecules/BenefitCardHeaderSummary.js:11]: There's a duplicated div here.

**Review by s-laugh — COMMENTED** [review]
Can you also merge dev into this, will fix the task lists, so we can verify they work nicely with this design update. Other than that it's great!

File: components/molecules/BenefitCardHeaderSummary.js (hunk 1 above)
Comment (shewood) [reply components/molecules/BenefitCardHeaderSummary.js:11]: Yes, I saw that and i will update. Thanks

File: components/molecules/BenefitCardHeaderSummary.js [hunk 2]
```diff
@@ -7,12 +7,12 @@ export default function BenefitCardHeaderSummary(props) {
   const typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
 
   return (
-    <div>
-      <p>{t[props.summary.type].title}</p>
-      <p className="font-bold">{props.summary.value}</p>
-      <p className="font-bold">{props.summary.status ?? null}</p>
+    <div className="w-full">
+      <p className="">{t[props.summary.type].title}</p>
```
Comment (DaceyTom2) [inline components/molecules/BenefitCardHeaderSummary.js:11]: className can be removed

**Comment by s-laugh** [comment]
Don't forget about the expanded section of the card - still needs some work.
Also be careful about editing those shared components, the text on some of the view more button looks off.
image

**Review by DaceyTom2 — APPROVED** [review]
Small change but looks great

**Comment by shewood** [comment]
I will take a look at the expanded section. I wasn't looking at that, just the header for now.

**Comment by s-laugh** [comment]
> I will take a look at the expanded section. I wasn't looking at that, just the header for now.

It's more the expanded section and view more were looking good in dev, so something is conflicting in the styles here.

File: components/molecules/BenefitCardHeaderSummary.js (hunk 2 above)
Comment (shewood) [reply components/molecules/BenefitCardHeaderSummary.js]: fixed

**Comment by shewood** [comment]
I updated the placement of the taskList. Now it shows under the button as expected.

**Review by s-laugh — APPROVED** [review]
Looks Good!

**Pull request merged**